crates/store/re_types/definitions/rerun/components/colormap.fbs

```diff
@@ -52,4 +52,10 @@ enum Colormap: ubyte {
     /// It is especially suited for visualizing signed values.
     /// It interpolates from cyan to blue to dark gray to brass to yellow.
     CyanToYellow,
+
+    /// The Spectral colormap from Matplotlib.
+    ///
+    /// This is a diverging colormap with cool colors on the ends and warm colors in the middle.
+    /// It interpolates from dark blue to light blue to green to yellow to orange to dark red.
```

Comment:
The image you shared on Slack suggests something else:

<img width="391" height="99" alt="image" src="https://github.com/user-attachments/assets/f4abffc9-f271-45f7-a8b9-0d94f45b23e3" />


```suggestion
    /// As it name implies, it follows the color spectra, like the colors in a rainbow:
    /// ROYGBIV (Red, Orange, Yellow, Green, Blue, Indigo, Violet)
```